### PR TITLE
Show a nice table when verifying OTA package

### PR DESCRIPTION
**Pull request by niclimcy**

### CODE REVIEW

File: src/css/table.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    background-color: #2c3034;
+}
+
+th {
+    background-color: #CCE8E9;
+    color: #000;
+}
+
+#app.dark th {
+  background: #324B4C;
```
Comment (luk1337) [inline src/css/table.css]: four spaces

File: src/components/verify-main/VerifyTabPage.vue [hunk 2]
```diff
@@ -1,60 +1,118 @@
...
+        <th colspan="2" :class="{ 'bg-red': !isVerified }">
+          {{ verifyResult }}
+        </th>
+      </tr>
+      <tr>
+        <td>File Name</td>
+        <td>{{ fileName }}</td>
+      </tr>
+      <tr v-if="verifySignInfo.commonName">
+        <td>CommonName</td>
```
Comment (luk1337) [inline src/components/verify-main/VerifyTabPage.vue]: spaces pls

File: src/components/verify-main/VerifyTabPage.vue [hunk 3]
```diff
@@ -1,60 +1,118 @@
...
-        const result = await CryptoService.verifyPackage(await fileReader.result);
-        alert(result.msg);
-
-        const form = this.$refs.form;
-        form.reset();
-      };
-      fileReader.readAsArrayBuffer(blob);
-    },
+<script setup>
+import { ref } from 'vue'
```
Comment (luk1337) [inline src/components/verify-main/VerifyTabPage.vue]: Also, pls bring back `;` as that matches existing code style.

File: src/components/verify-main/VerifyTabPage.vue [hunk 4]
```diff
@@ -1,60 +1,118 @@
...
+      <tr v-if="verifySignInfo.stateOrProvinceName">
+        <td>StateOrProvinceName</td>
+        <td>{{ verifySignInfo.stateOrProvinceName }}</td>
+      </tr>
+      <tr v-if="verifySignInfo.countryName">
+        <td>CountryName</td>
+        <td>{{ verifySignInfo.countryName }}</td>
+      </tr>
+      <tr v-if="verifySignInfo.publicKeyFingerprint">
+        <td>Public Key</td>
```
Comment (luk1337) [inline src/components/verify-main/VerifyTabPage.vue]: Public Key Fingerprint?

File: src/components/verify-tab/VerifyTabPage.vue [hunk 5]
```diff
@@ -1,13 +1,62 @@
...
+      <tr v-if="verifySignInfo.organizationName">
+        <td>Organization</td>
+        <td>{{ verifySignInfo.organizationName }}</td>
+      </tr>
+      <tr v-if="verifySignInfo.localityName">
+        <td>Locality</td>
+        <td>{{ verifySignInfo.localityName }}</td>
+      </tr>
+      <tr v-if="verifySignInfo.stateOrProvinceName">
+        <td>State Or ProvinceName</td>
```
Comment (luk1337) [inline src/components/verify-tab/VerifyTabPage.vue]: lowercase `or`

File: src/components/verify-tab/VerifyTabPage.vue [hunk 6]
```diff
@@ -1,13 +1,62 @@
...
+        <td>{{ verifySignInfo.stateOrProvinceName }}</td>
+      </tr>
+      <tr v-if="verifySignInfo.countryName">
+        <td>Country Name</td>
+        <td>{{ verifySignInfo.countryName }}</td>
+      </tr>
+      <tr v-if="verifySignInfo.publicKeyFingerprint">
+        <td>Public Key</td>
```
Comment (luk1337) [inline src/components/verify-tab/VerifyTabPage.vue]: ...Fingerprint?

File: src/components/verify-tab/VerifyTabPage.vue [hunk 7]
```diff
@@ -1,13 +1,62 @@
 <template>
   <div
-      class="tab-page verify-tab-page"
-      v-on:dragover="fileDragOver"
-      v-on:dragleave="fileDragLeave"
-      v-on:drop="fileDropped"
+    class="tab-page verify-tab-page"
```
Comment (luk1337) [inline src/components/verify-tab/VerifyTabPage.vue]: 2 more spaces

File: src/components/verify-tab/VerifyTabPage.vue [hunk 8]
```diff
@@ -17,33 +66,46 @@ import CryptoService from '../../js/CryptoService';
...
-        form.reset();
+        const result = await CryptoService.verifyPackage(fileReader.result);
+        this.isVerified = result.status;
+        this.verifyResult = result.msg;
+        this.verifySignInfo = result.signInfo;
       };
       fileReader.readAsArrayBuffer(blob);
+      this.fileName = blob.name;
+    },
+    verifyFileInput(event) {
```
Comment (luk1337) [inline src/components/verify-tab/VerifyTabPage.vue:107]: mismerge

File: src/components/verify-tab/VerifyTabPage.vue (hunk 7 above)
Comment (niclimcy) [reply src/components/verify-tab/VerifyTabPage.vue]: wait why?

Comment (luk1337) [reply src/components/verify-tab/VerifyTabPage.vue]: Because other files do that.

Comment (luk1337) [reply src/components/verify-tab/VerifyTabPage.vue]: e.g. that one https://github.com/lineageos-infra/updater-client/blob/main/src/components/changes-tab/ChangesGroup.vue

Comment (niclimcy) [reply src/components/verify-tab/VerifyTabPage.vue]: Hmm interesting codestyle... ok

**Pull request merged**